.circleci/config.yml

```diff
@@ -784,6 +784,19 @@ jobs:
       - react-get-deps
       - run: make -C webui/react check
 
+  build-react-preview:
+    docker:
+      - image: cimg/node:12.16
+    environment:
+      PUBLIC_URL: /0
```

Comment:
**question:** should this point to something like `http://latest-master.determined.ai:8080'? Or what server should CI be communicating with?

Reply:
this var will stay the same but we will set the `SERVER_URL` to what you pointed out after this pr has landed and latest-master is updated.